src/AzureAuth/CommandMain.cs

```diff
@@ -237,6 +237,7 @@ public EventData AuthFlowEventData(AuthFlowResult result)
             var eventData = new EventData();
             eventData.Add("authflow", result.AuthFlowName);
             eventData.Add("success", result.Success);
+            eventData.Add("duration_milliseconds", result.DurationInMs);

```

Comment:
To keep the telemetry values clean - we can cast the total ms `long` to an `int`. That's close enough accuracy on the runtime for this.